### PR TITLE
Add menu in lunch

### PR DESCRIPTION
**Pull request by wave49192**
- add lunch menu in `Menu.md`
- add a link to your menu section in `README.md`

### CODE REVIEW

**Comment by Siraphop4Nene** [comment]
Good work!!

**Review by Siraphop4Nene — APPROVED** [review]

**Review by sittipat-chandang — CHANGES_REQUESTED** [review]
Please rearrange the Lunch menu into an alphabetical order.

File: Menu.md [hunk 1]
```diff
@@ -6,7 +6,13 @@
...
     (use proper Markdown formatting, not literal text like this)
 
-## Lunch
+## Lunch 
+| Item| Price  |
+|---|:----------|
+| Hamburger | 50 Baht |
+| Fried Rice | 25 Baht|
+| Noodles | 30 Baht    |
+| Fried Chicken | 40 Baht     |
```
Comment (sirateek) [inline Menu.md]: The menu should be alphabetically ordered

**Review by sittipat-chandang — APPROVED** [review]

**Review by NamoSmith — APPROVED** [review]
Nice work!

**Pull request merged**